### DISCUSSION TITLE
Support for gRPC JSON Transcoding with Scalar.AspNetCore in .NET 9?

### DISCUSSION THREAD

**Discussion by python3js** [scalar/scalar#3181] [Q&A] [answered]
I’m currently using .NET 9, which has built-in support for generating OpenAPI documents, and I was wondering if Scalar.AspNetCore supports rendering API references for gRPC services with JSON transcoding. If not, are there any plans to add support for this in the future?

**Comment by marclave** [comment, marked as answer]
hey @python3js !

we are looking to add grpc support to scalar #2683

**Reply by Themba-01** [reply]
Hi, have you added it?
